src/drt/src/dr/FlexDR_maze.cpp

```diff
@@ -2787,6 +2800,33 @@ void FlexDRWorker::setNDRStyle(drNet* net,
   }
 }
 
+void FlexDRWorker::editStyleExt(frSegStyle& currStyle,
+                                frMIdx startX,
+                                frMIdx endX,
+                                frMIdx startY,
+                                frMIdx endY,
+                                frMIdx z,
+                                FlexMazeIdx* prev,
+                                FlexMazeIdx* next)
+{
+  auto layer = getTech()->getLayer(gridGraph_.getLayerNum(z));
+  if (layer->getWrongDirWidth() >= layer->getWidth())
+    return;
```

Comment:
Why?

Reply:
this function clips the right way segment extension if the wrongway segment widht is smaller to avoid minstep violations. if wrongway width is larger than default width, then the wrongway segment if present is already covering the extension of the rightway segment. so no need for clipping

Reply:
if the wrongway width is large then couldn't it's 1/2 width extension go beyond the narrower wire?

Reply:
the wrongway segment has the same extension as the rightway segment, half of the default width. So it wouldn't be a problem.

Reply:
I'm thinking of:
![image](https://user-images.githubusercontent.com/761514/193067451-22139e86-1d73-4c22-8056-654e6b2c47fa.png)

Reply:
That would never happen because the wrongway segment would never go beyond the rightway segment. The wrongway segment's extension is half of the default width not the wrongway width.
![wrongway drawio (1)](https://user-images.githubusercontent.com/56893454/193463673-e8fc34ba-60c9-4b01-83ec-ca133f9cc58f.png)

Reply:
No the default extension is 1/2 the wire width not the default width:
![image](https://user-images.githubusercontent.com/761514/193469016-592cb335-3263-45f7-a19d-e92fa20303b5.png)